client/src/components/TripChecklist.vue

```diff
@@ -213,13 +213,15 @@ async function addItem(itemName) {
     return
   }
 
+  const itemPosition = (items.value.slice(-1)[0]?.index || 0) + 1_000_000
```

Comment:
Det føles litt rart å sette den indeksen client side, but it's OK maybe??

Reply:
Øhhh, enig - men index var allerede satt client side så da fortsatt jeg bare med det?